linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_server.js

```diff
@@ -1,28 +1,106 @@
 var RouterServer = (function() {
-  // Skeletal implementation to demonstrate functionality of RouterController
+  var template;
 
-  function initializeServerContainers(servers, $serverEl) {
-    _.each(servers, function(server) {
-      var $se = $("<div />").addClass("router-server")
-        .html("<div>Servers</div><div>" + server.prefix + "</div>");
-      $se.appendTo($serverEl);
+  function renderServer($container, server, data) {
+    var serverHtml = template({
+      server: server.label,
+      metrics: data
     });
-  }
+    var $serverHtml = ($("<div />").addClass("router-server").html(serverHtml));
 
-  function renderServers() {
+    $container.html($serverHtml);
```

Comment:
I'm a little confused by this. If I'm understanding correctly, $container is a `<div class="router-server"/>` but you then stick another `<div class="router-server"/>` into it .... thereby recursively nesting this div on every update?